omod/src/main/java/org/openmrs/module/bahmni/ie/apps/model/BahmniFormData.java

```diff
@@ -3,15 +3,15 @@
 import java.util.List;
 
 public class BahmniFormData {
-    private BahmniForm bahmniForm;
+    private BahmniForm formJson;
```

Comment:
bahmniForm looks more meaningful here? BahmniForm doesn't hold any json data

Reply:
Frontend (Implementer-interface) will be using this object to convert into a json file. When the name is mismatching for download, the import will be failing for the same json file. That is the reason why we reverted back to formJson.

Reply:
So it was formJosn, then renamed to bahmniForm then for the above reason you reverted back?

Reply:
yes @shivarach We thought "bahmniForm" would make more sense but as @AlekhyaYalla mentioned, from UI we are expecting it as "formJson". Changing the object name now as part of "export" feature would effect the existing forms naming convention and import of the form may fail.